about.html

```diff
@@ -0,0 +1,133 @@
+      <p class="about-contact-mail">hernanguemes@hotmail.com</p>
+    </div>
+  </section>
+
+ 
+<!-- --------------- LOGO Section ----------------- -->
+
+  <section class="logo-section">
+    <div class="logo-creation">
+      <h2 class="logo-creation-title">GEF & Five Percent Logos</h2>
+      <div class="separator"></div>
+      <p>GEF was created in 2023 for the event. 5 percent per Nature Logo belongs to The Orvis Company. Orvis pioneered the coporate conservation support in the 1980s.</p>
+    </div>
+    <div class="forum-logo-containers">
+      <img class="forum-logo2" src="./Images/GEF 300 px.png" alt="Global Conservation Forum">
+      <a href="https://www.orvis.com/impact.html" target="blank"><img class="five-per-nature-logo2" src="./Images/Five-percent per Nature.svg" alt="Five percent per Nature Logo"></a>
+    </div>
+  </section>
+
+  <!-- ----------------- Impact Projects Section ----------------- -->
+
+  <section class="review-impact-container">
+      <div class="impact-detail-container">
+        <h2>Take a look at some restoration projects</h2>
+        <div class="separator"></div>
+        <h3>Review some of the projects run by some of the sponsor companies</h3>
+      </div>
+     
+      <div class="example-past-project-container">
+        <div id="tdf" class="bg-position-style ">
+          <div>
+            <h2>2005</h2>
+            <p>Tierra del fuego, Fin del Mundo, Argentina</p>            
+          </div>  
+        </div>
+        <div id="malleo" class="bg-position-style ">
+          <div>
+            <h2>2010</h2>
+            <p>Malleo River, Argentine Patagonia</p>
+          </div>
+        </div>
+        
+  </section>
+
+  </main>
+
+  <!-- --------------------- <SPONSORS SECTION> --------------------- -->
+ 
+  <footer>
+    <div class="sponsor-section-container">
+      <p class="sponsor-title mobile">Sponsors</p>
+      <div class="separator mobile"></div>
+      <div class="sponsor-logos-container mobile">
+        <img class="sponsor-logo" src="./Images/nature sin fondo.png" alt="Main Sponsor Orvis Company, 5 percent for Nature Logo">
+        <img class="sponsor-logo" src="./Images/National Parks Asociation.jpg" alt="Sponsor National Parks Asociation Logo">       
+        <img class="sponsor-logo fd" src="./Images/flyDreamers Logo.png" alt="Main Sponsor Fly Dreamers Logo">
+        <img class="sponsor-logo" src="./Images/Parques Nacionales Argentina.png" alt="Sponsor National Parks from Argentina logo">
+        <img class="sponsor-logo" src="./Images/Patagonia.png" alt="Sponsor Patagonia Brand Logo">
+      </div>
+    </div>
+
+    <div class="legal-rights2">
+      <img src="./Images/Forum Logo.png" alt="Global forum Logo"/>
+      <p>Original Idea by Cindy Shin for Creative Commons Korea 2015. Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Please consider increasing the height of the div a bit. Also kindly make sure to reduce the same between logo and text 🤓⛏ (Home page footer looks good)

![image](https://github.com/HFG43/Capstone1/assets/106119105/ba19cbfc-7621-4fa5-9549-752c22c7eb4f)